football-manager/src/lib/Pages/Club/ClubCalendar/ClubCalendar.svelte

```diff
@@ -1,7 +1,26 @@
 <script>
     export let id
+    let games = [];
     import DisplayCalendar from "../../../Components/DisplayCalendar.svelte";
     import ClubNavBar from "../clubNavBar.svelte";
+
+    const fetchGames = fetch(`http://localhost:8080/club/${id}/games`)
+    .then((res) => {
+      return res.json();
+    })
+    .then((data) => {
+      console.log(data);
+      if (Array.isArray(data.games)) {
+        games = data.games;
+        console.log(games)
+      }
```

Comment:
```suggestion
        games = data
        console.log(games)
```
Check postman, request only contains one array, not the a game object